features/web-cryptography.yml

```diff
@@ -5,8 +5,22 @@ spec:
   - https://wicg.github.io/webcrypto-secure-curves/
 caniuse: cryptography
 group: security
+# Override the status to match caniuse. This is the same as one would get by
+# computing the status from api.SubtleCrypto.encrypt/decrypt/sign/verify, except
+# for Edge 12-18 where BCD has partial_implementation due to some unsupported
+# algorithms. However, the supported algorithms have changed over time, so it
+# seems more fair to treat it as supported in Edge 12-18.
```

Comment:
I filed https://github.com/mdn/browser-compat-data/issues/27669 for this. It seems like BCD ought to break out all the algorithms as separate subfeatures. Not sure how much, if at all, you wish to prioritize such a clean up.

Reply:
Good idea to break it into subfeatures, that would help us here. But doing so would require confirming support in all browsers, which would be a lot of work...